doc/source/math/basic.rst

```diff
@@ -294,3 +295,12 @@ Trigonometric Functions
         PRINT ARCTAN2(0.67, 0.89). // prints 36.9727625
 
     The two parameters resolve ambiguities when taking the arctangent. See the `wikipedia page about atan2 <http://en.wikipedia.org/wiki/Atan2>`_ for more details.
+
+.. function:: CHAR(a)
```

Comment:
This not only puts it in math which is kinda close. but in trig functions which i dont think we want at all. ill move it up into the basic math stuff